galasa-parent/dev.galasa.framework.api.cps/src/main/java/dev/galasa/framework/api/cps/internal/routes/CPSRoute.java

```diff
@@ -170,6 +177,23 @@ protected  boolean checkNamespaceExists(String namespaceName) throws Configurati
         return valid;
     }
 
+    /**
+     * Sort the properties provided by key 
+     * @param properties
+     * @return Sorted Map of properties
+     */
+    protected Map<GalasaPropertyName, CPSProperty> sortResults(Map<GalasaPropertyName, CPSProperty> properties){
```

Comment:
`sortResults` is a bit confusing - maybe `sortPropertiesByKey` or something better?